.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Docs
+
+on:
+  push:
+    branches:
```

Comment:
this should run on prs too, but just not update the gh-pages branch. See https://github.com/mrc-ide/hipercow/blob/main/.github/workflows/pkgdown.yaml for how we do this on R builds.

Reply:
done